src/features/swap/helpers/data/stablecoins.ts

```diff
@@ -0,0 +1,188 @@
+export const stableCoinAddresses: {
+  [address: string]: {
+    name: string
+    symbol: string
+    chain: 'gnosis' | 'mainnet' | 'arbitrum' | 'sepolia'
+  }
+} = {
+  '0xdd96b45877d0e8361a4ddb732da741e97f3191ff': {
+    name: 'BUSD Token from BSC',
+    symbol: 'BUSD',
+    chain: 'gnosis',
```

Comment:
nit: maybe it is easier to maintain and less redundant if you use a `Record<Chain, Token[]>`?

Reply:
yes! The reason I did it like this is that I didn't want to pass the chainId/chainName. 
Also, name, symbol, chain - not necessary actually. We should maybe just go with token addresses.